### PR TITLE
subst: add page

### PR DESCRIPTION
**Pull request by yanuehara**
----

- [X] The page (if new), does not already exist in the repo.

- [X] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [X] The page has 8 or fewer examples.

- [X] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [X] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/windows/subst.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# subst
+
+> Associates a path with a drive letter.
```
Comment (agnivade) [inline pages/windows/subst.md]: Maybe better to say "Associates a path with a virtual drive letter.". To better signify that these are virtual drive mappings and not actual physical drives.

File: pages/windows/subst.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+- List active substitutions:
```
Comment (agnivade) [inline pages/windows/subst.md]: How about "List active associations" ? To match with the rest of the examples.

File: pages/windows/subst.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+`subst`
+
+- Add an association:
+
+`subst {{drive_letter_with_colon}} {{path/to/folder}}`
```
Comment (agnivade) [inline pages/windows/subst.md]: Better to give a concrete example with "Z:". Something like - `subst {{Z:}} {{C:\User\Python2.7}}`

File: pages/windows/subst.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+- Remove an association:
+
+`subst {{drive_letter_with_colon}} /d`
```
Comment (agnivade) [inline pages/windows/subst.md]: Same as above. Use `Z:`

**Comment by sbrl** [comment]
Perhaps we should put together one of those lists like we did with that list of 10 cli alternatives? Like just a shortish list to get people started.

**Comment by agnivade** [comment]
Umm sure. Like a windows list you mean ? I am not too familiar with windows though. Feel free to make one.

**Comment by yanuehara** [comment]
I've made the corrected suggestions and re-pushed the commit

**Comment by agnivade** [comment]
Thanks. You need not amend to the commit and force push. You can have a separate commit and push that. It helps to see what changes have happened during a review and to compare between reviews.

While merging, there is a squash button which allows us to squash all commits to one.

**Comment by sbrl** [comment]
Thanks, @yanuehara! Never knew this existed :P

**Comment by yanuehara** [comment]
Thanks @agnivade I didn't know that, I'll be careful next contributions. @sbrl It's useful e.g when you need to emulate an CD driver using the iso file uncompressed.

**Pull request merged**